docs/src/visualize.md

```diff
@@ -0,0 +1,17 @@
+# Visualize output
+
+In the examples we use [Makie.jl](https://makie.juliaplots.org/stable/) for plotting.
+
+Makie comes with few [backends](https://makie.juliaplots.org/stable/#makie_ecosystem). In the documented examples
+we use [CairoMakie](https://makie.juliaplots.org/stable/documentation/backends/cairomakie/) since this backend
+works well on headless devices, that is, devices without monitor. Since the documentation is automatically
+build via GitHub actions the CairoMakie backend is necessary. Users that run GeophysicalFlows.jl on
```

Comment:
build -> built

GeophysicalFlows.jl -> `GeophysicalFlows.jl` maybe even with the link?